terraform/helm/genesis/files/genesis.sh

```diff
@@ -41,11 +41,17 @@ echo "FULLNODE_INTERNAL_HOST_SUFFIX=${FULLNODE_INTERNAL_HOST_SUFFIX}"
 echo "STAKE_AMOUNT=${STAKE_AMOUNT}"
 echo "NUM_VALIDATORS_WITH_LARGER_STAKE=${NUM_VALIDATORS_WITH_LARGER_STAKE}"
 echo "LARGER_STAKE_AMOUNT=${LARGER_STAKE_AMOUNT}"
+echo "RANDOM_SEED=${RANDOM_SEED}"
+
+RANDOM_SEED_IN_DECIMAL=$(printf "%d" 0x${RANDOM_SEED})
 
 # generate all validator configurations
 for i in $(seq 0 $(($NUM_VALIDATORS-1))); do
     username="${USERNAME_PREFIX}-${i}"
     user_dir="${WORKSPACE}/${username}"
+    seed=$(printf "%064x" "$((${RANDOM_SEED_IN_DECIMAL}+i))")
```

Comment:
nit: move these lines into the below conditional for `RANDOM_SEED`. Otherwise we're generating the seed and printing it out, but not using it, which will look confusing from the logs